### PR TITLE
[PS-42886] Adding method to handle payment activity's onResume instead of exposing callbacks directly

### PR DESCRIPTION
**Pull request by pdrlima**

### CODE REVIEW

**Review by RamyaAdattiri — APPROVED** [review]
👍

File: android/build.gradle [hunk 1]
```diff
@@ -46,7 +46,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.facebook.react:react-native:0.59.10'
+    implementation 'com.facebook.react:react-native:0.61.5'
```
Comment (RamyaAdattiri) [inline android/build.gradle:49]: Are we changing to this version in sdk ?

**Review by millibeckers — APPROVED** [review]
👍

File: android/build.gradle (hunk 1 above)
Comment (pdrlima) [reply android/build.gradle:49]: we are indeed

Comment (pdrlima) [reply android/build.gradle:49]: it actually already was, this library was just outdated. Soon we will come back to update this to 65

**Pull request merged**